File: PasswordManagerUI/PasswordManager/Account.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PasswordManager
{
    /**
     * This class is specifically used to parse the password json file
     * Each account entry will have a site, username, email, password, and 
     * other is just used for anything else like security questions 
     */
    class Account
    {
        public string Site { get; set; }
        public string Username { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public string Other { get; set; }

        public Account(string site, string username, string email, string password, string other)
        {
            this.Site = site;
            this.Username = username;
            this.Email = email;
            this.Password = password;
            this.Other = other;
        }
        public Account()
        {
            this.Site = "";
            this.Username = "";
            this.Email = "";
            this.Password = "";
            this.Other = "";
        }
    }
}

File: password_manager/lib/models/account.dart
import 'package:flutter/material.dart';

class Account {
  final String site;
  final String username;
  final String email;
  final String password;
  final String other;

  Account({
    required this.site,
    required this.username,
    required this.email,
    required this.password,
    required this.other,
  });

  factory Account.fromJson(Map<String, dynamic> json) {
    return Account(
      site: json['Site']?.trim() ?? '',
      username: json['Username']?.trim() ?? '',
      email: json['Email']?.trim() ?? '',
      password: json['Password']?.trim() ?? '',
      other: json['Other']?.trim() ?? '',
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'Site': site,
      'Username': username,
      'Email': email,
      'Password': password,
      'Other': other,
    };
  }

  String toString() {
    return 'Account(site: $site, username: $username, email: $email, password: $password, other: $other)';
  }
}
